src/components/FileTree/index.spec.tsx

```diff
@@ -42,56 +40,51 @@ describe(__filename, () => {
     });
 
     it('converts a directory entry to a directory node', () => {
-      const versionId = 'some-name';
+      const versionId = '1234';
 
-      const directory = 'some-directory';
+      const filename = 'some-directory';
       const entries = [
-        {
-          depth: 0,
-          type: VersionEntryType.directory,
-          filename: directory,
-          path: directory,
-          modified: '2019-01-01',
-        },
+        createInternalVersionEntry({
+          ...fakeVersionEntry,
+          directory: true,
+          filename,
+        }),
       ];
 
-      const data = buildFileTree(versionId, entries);
+      const tree = buildFileTree(versionId, entries);
 
-      expect(data.children).toEqual([
+      expect(tree.children).toEqual([
         {
           id: entries[0].path,
-          name: directory,
+          name: filename,
           children: [],
         },
       ]);
     });
 
     it('finds the appropriate node to add a new entry to it', () => {
-      const versionId = 'some-name';
+      const versionId = '1234';
 
       const directory = 'parent';
       const file = 'child';
 
       const entries = [
-        {
-          depth: 0,
-          type: VersionEntryType.directory,
+        createInternalVersionEntry({
+          ...fakeVersionEntry,
+          directory: true,
           filename: directory,
-          path: directory,
-          modified: '2019-01-01',
-        },
-        {
+        }),
+        createInternalVersionEntry({
+          ...fakeVersionEntry,
           depth: 1,
```

Comment:
As above, maybe include `directory: false` to be explicit?

Reply:
good call